INSTALL.md

```diff
@@ -281,8 +280,7 @@ in `/usr/include/sys/dlpi.h`, and find the corresponding value.
 	pcap-dbus.c	    - D-Bus capture support
 	pcap-dbus.h	    - D-Bus capture support
 	pcap-dlpi.c	    - Data Link Provider Interface support
-	pcap-dos.c	    - MS-DOS capture support
-	pcap-dos.h	    - headers for MS-DOS capture support
+	pcap-dos.c	    - removed in 2023, after libpcap 1.10
```

Comment:
Should we do that for the files removed in the past?

And should the list of "were here, but aren't here any more" files be a separate list?

Reply:
Only if there is a good reason, it seems to me. For example, files in `missing/` come and go, but the world does not need to know so long as the source compiles and works correctly.